### PR TITLE
Bito nutshell0

### PR DESCRIPTION
**Pull request by bitomann**
1. git fetch --all
2. git checkout bito-nutshell0

I had edit and delete working but now I keep getting an undefined error when I click the "Submit" button. Would you guys mind seeing if you can see why?

### CODE REVIEW

**Comment by CoopNich** [comment]
Refer to what I said in slack.  You'll want to reference the right variable within your "if" conditional.

Also your main.js seems to have not merged properly.

**Comment by MReeds** [comment]
I agree with Cooper. Give that a shot before merging

File: src/scripts/messages/events.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+        submitButton.addEventListener("click", (event) => {
+            event.preventDefault()
+            const messageInput = document.querySelector("#messageInput");
+
+            const message = {
+                message: messageInput.value
+            };
+
+            if (messageId.value !== "") {
```
Comment (CoopNich) [inline src/scripts/messages/events.js:28]: Right here.

**Review by CoopNich — CHANGES_REQUESTED** [review]
I left a comment on the line of code I'm talking about

**Pull request closed without merging**